packages/permission-controller/package.json

```diff
@@ -33,9 +33,9 @@
     "@metamask/approval-controller": "^4.1.0",
     "@metamask/base-controller": "^3.2.3",
     "@metamask/controller-utils": "^5.0.2",
-    "@metamask/json-rpc-engine": "^7.1.1",
+    "@metamask/json-rpc-engine": "^7.2.0",
     "@metamask/rpc-errors": "^6.1.0",
-    "@metamask/utils": "^8.2.0",
```

Comment:
Why is this package being downgraded?

Reply:
Thanks for catching this! I just pushed a fix. 

I'll add an entry in the migration checklist to double-check for dependency version bumps or other changes in main that may have happened while the PR is open. 

For future migrations, we plan to coordinate to minimize the time these PRs are kept open so there's less chance of something like this slipping through.